behat-tests/features/acquia_lift_element_variations.feature

```diff
@@ -1,170 +1,198 @@
-#Assumes Marketer role.
+@api @javascript @insulated @campaign
 Feature: Personalize elements variations can be edited for an existing campaign.
   In order to manage element variations in context
   As a site marketer
   I want the ability to add, edit, and delete existing personalize element variations in context.
 
-  @api @javascript @campaign
   Scenario: Add personalize elements to a campaign
+    # I have a campaign.
+    # I login with the marketer role.
+    # I am on an article page.
     Given "acquia_lift" agents:
-      | machine_name                    | label |
+      | machine_name                    | label                           |
       | testing-campaign-add-variations | Testing campaign add variations |
-    And I am logged in as a user with the "Marketer" role
+    And I am logged in as a user with the "access administration pages,access toolbar,administer visitor actions,manage personalized content" permission
```

Comment:
Permission is not set up by a programmer, but by a Drupal site admin in site's UI. This site admin is not necessarily a programmer, and is often not. This means Permission is a really Drupal feature, and we are really saying "before using the Lift feature, the user needs a set up in the Permission feature".

We can always introduce Roles as the collection of permissions, but the argument falls back to the fact that we will still have to somehow clarify at each use that the Role "marketer" is "those four specific permissions and nothing more", so we are not really cutting any maintenance here.